docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,142 @@
+Suggesting Apps for Split Tunneling
```

Comment:
This document is missing the header components defined in the template. See: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/docs/rfcs/0001-rfc-process.md#guide-level-explanation.

Reply:
Also, the PR title is missing the `[RFC]` prefix as defined in the same document I linked before.

Reply:
Updated the title, thanks. I think I'm missing the reference in the doc to "header components" - sorry to be dense, but can you point out what you mean? (What I do see on the doc is: `I'd prefer we not be overly prescriptive about it, at least early on.`)

Reply:
Hm, you are right. I seem to remember from the discussion on that first RFC that the first RFC itself would be used as a template, but if it's not in the doc it's not 🤷 I meant the list of stuff right at the top, but feel free to ignore.